docs/configuration.md

```diff
@@ -0,0 +1,457 @@
+>  preferNative: PKPreferNativeConfigObject
+>}
+>```
+>##### Default: 
+>```js
+>{
+>  audioLanguage: "",
+>  textLanguage: "",
+>  useNativeTextTrack: false,
+>  volume: 1,
+>  playsinline: true,
+>  preload: "none",
+>  autoplay: false,
+>  allowMutedAutoPlay: true,
+>  muted: false,
+>  options: {
+>    html5: {
+>      hls: {},
+>      dash: {}
+>    }
+>  },
+>  preferNative: {
+>    hls: false,
+>    dash: false
+>  },
+>  streamPriority: [
+>    {
+>      engine: "html5",
+>      format: "hls"
+>    },
+>    {
+>      engine: "html5",
+>      format: "dash"
+>    },
+>    {
+>      engine: "html5",
+>      format: "progressive"
+>    }
+>  ]
+>}
+>```
+>##### Description: Defined the playback options.
+>>### config.playback.audioLanguage
+>>##### Type: `string`
+>>##### Default: `""`
+>>##### Description: Defined default audio track language.
+>>If audio track with the defined language is exists, this audio track will be selected as the initial audio track.
+>> #### Example:
+>>```js
+>>var config = {
+>>    playback: {
+>>        audioLanguage: "eng" // Start playback with english audio
+>>    }
+>>};
+>>```
+>## 
+>>### config.playback.textLanguage
+>>##### Type: `string || "auto"`
+>>##### Default: `""`
+>>##### Description: Defined default captions language.
+>>If captions with the defined language is exists, this text track will be selected as the initial text track.
+>> #### Example:
+>>```js
+>>var config = {
+>>    playback: {
+>>        textLanguage: "heb" // Start playback with hebrew captions
+>>    }
+>>};
+>>```
+>> If the value `"auto"` is set, i.e:
+>>```js
+>>var config = {
+>>    playback: {
+>>        textLanguage: "auto"
+>>    }
+>>};
+>>```
+>> The player will choose default captions language by the following logic:
+>> 1. **Locale language** - If there are captions in the user's system language then this language will be selected.
+>> 2. **Manifest default language** - If a default language is specified in the manifest file then this language will be selected.
+>> 3. **First language in manifest** - The first language specified in the manifest file will be selected.
+>> 4. If none of the above conditions have taken place, do not display captions.
+>## 
+>>### config.playback.useNativeTextTrack
+>>##### Type: `boolean`
+>>##### Default: `false`
+>>##### Description: Whether to use native browser text tracks.
+>>If sets to true, native browser captions will be displayed. 
+>## 
+>>### config.playback.volume
+>>##### Type: `number`
+>>##### Default: `1`
+>>##### Description: Defined the initial volume value.
+>>Value must be in the range of 0-1.
+>> #### Example:
+>>```js
+>>var config = {
+>>    playback: {
+>>        volume: 0.5
+>>    }
+>>};
+>>```
+>## 
+>>### config.playback.playsinline
+>>##### Type: `boolean`
+>>##### Default: `true`
+>>##### Description: A Boolean attribute which indicates that the video is to be played "inline", that is within the element's playback area.
```

Comment:
**Description: A Boolean attribute that indicates whether the video should be played "inline", that is, within the element's playback area. This is especially relevant when playing videos on iPhone devices, where - if the value is set to false - the video will be played using the AV Player (iOS native video player).**